src/runtime/lpython_builtin.py

```diff
@@ -288,3 +288,12 @@ def lbound(x: i32[:], dim: i32) -> i32:
 
 def ubound(x: i32[:], dim: i32) -> i32:
     pass
+
+
+@overload
+def imag(x: c64) -> f64:
```

Comment:
This is not a built-in function, right? Maybe add a leading underscore.

Reply:
Done!